jta-and-hibernate-standalone/src/main/java/org/jboss/narayana/quickstart/jta/TransactionalConnectionProvider.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
All the quickstars I saw in WildFly and most of our quickstarts have the copyrights in them.